docs/resources/conversation.md

```diff
@@ -68,6 +87,15 @@ conversation should be archived or left behind on destroy. Valid values are
 `archive | none`. Note that when set to `none` the conversation will be left
 as it is  and as a result any subsequent runs of terraform apply with the same
 name  will fail.
+- `adopt_existing_channel` (Optional, Default `false`) indicates that an existing 
+channel with the same name should be adopted by terraform and put under state
+management. If the existing channel is archived, it will be unarchived. (Note: for 
+unarchiving of existing channels to work correctly, you _must_ use a user token,
+not a bot token, due to bugs in the Slack API)
+- `remove_non_members` (Optional, Default `true`) indicates that members of the
+channel that aren't listed in the `permament_members` array will be kicked. (Note:
+your Slack configuration may restrict/prohibit kicking of members from channels
+except when using administrator credentials)
```

Comment:
this is especially needed for adopting existing channels, as permission to remove non members may be denied by slack configuration.